android crashes reported in android play

Mutiple crashes with minor differences in line number

```
Caused by: java.lang.RuntimeException: 
 
  at edu.berkeley.eecs.emission.cordova.transitionnotify.TransitionNotificationReceiver.getFirstLocation (TransitionNotificationReceiver.java:279)
  at edu.berkeley.eecs.emission.cordova.transitionnotify.TransitionNotificationReceiver.getTripStartEndData (TransitionNotificationReceiver.java:238)
  at edu.berkeley.eecs.emission.cordova.transitionnotify.TransitionNotificationReceiver.fireGenericTransition (TransitionNotificationReceiver.java:107)
  at edu.berkeley.eecs.emission.cordova.transitionnotify.TransitionNotificationReceiver.onReceive (TransitionNotificationReceiver.java:88)
  at android.app.ActivityThread.handleReceiver (ActivityThread.java:3011)
```

```
Caused by: java.lang.RuntimeException: 
  at edu.berkeley.eecs.emission.cordova.transitionnotify.TransitionNotificationReceiver.getFirstLocation (TransitionNotificationReceiver.java:279)
  at edu.berkeley.eecs.emission.cordova.transitionnotify.TransitionNotificationReceiver.getTripStartEndData (TransitionNotificationReceiver.java:238)
  at edu.berkeley.eecs.emission.cordova.transitionnotify.TransitionNotificationReceiver.fireGenericTransition (TransitionNotificationReceiver.java:107)
  at edu.berkeley.eecs.emission.cordova.transitionnotify.TransitionNotificationReceiver.onReceive (TransitionNotificationReceiver.java:88)
  at android.app.ActivityThread.handleReceiver (ActivityThread.java:2743)
```

--- a/src/android/TransitionNotificationReceiver.java
+++ b/src/android/TransitionNotificationReceiver.java
@@ -275,8 +275,14 @@ public class TransitionNotificationReceiver extends BroadcastReceiver {
                     Log.d(context, TAG, "Found no locations before exiting geofence while ending trip!");
                     SimpleLocation firstLoc = getDefaultLocation(context);
                     if (firstLoc.getTs() < startTransition.getTs()) {
+                        if (BuildConfig.DEBUG) {
                         throw new RuntimeException("firstLocArray[0].ts "+firstLoc.getTs()
                             +" < startTransition.ts "+startTransition.getTs());
+                        } else {
+                            // Explanation for returning null at
+                            // https://github.com/e-mission/e-mission-transition-notify/issues/12#issuecomment-322535726
+                            return null;
+                        }
                     }
                     return firstLoc;
                 } else {

